hooks/hookstage/invocation.go

```diff
@@ -1,14 +1,89 @@
 package hookstage
 
-import "github.com/prebid/prebid-server/hooks/hookanalytics"
+import (
+	"encoding/json"
+	"sync"
+
+	"github.com/golang/glog"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/hooks/hookanalytics"
+)
+
+type Entity string
+
+const (
+	EntityHttpRequest              Entity = "http-request"
+	EntityAuctionRequest           Entity = "auction-request"
+	EntityAuctionResponse          Entity = "auction-response"
+	EntityAllProcessedBidResponses Entity = "all-processed-bid-responses"
+)
```

Comment:
As we understood from PBS-Java, the purpose of these constants is to provide additional information to the client regarding the type of object that was processed during the stage execution.
For most stages, the entity will be equal to one of the constants above. But for the `bidder-request` and `raw-bidder-response` stages, the `entity` is set to the bidder name.

We changed these constants to private since they are used only within this package.